shazam/shazam.py

```
@@ -0,0 +1,65 @@
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)